addons/website/static/src/less/website.ui.less

```diff
@@ -4,14 +4,16 @@
 
 // LAYOUTING
 body.o_connected_user {
-    padding-top: @odoo-navbar-height!important;
+    padding-top: @odoo-navbar-height - 2!important;
```

Comment:
Why `- 2`? Even if we remove the border, the navbar total height should still be `@odoo-navbar-height`